docs/content/preview/integrations/atlas-ycql.md

```diff
@@ -0,0 +1,163 @@
+    #Hbase
+    #atlas.graph.storage.hostname=localhost
+    #atlas.graph.storage.hbase.regions-per-server=1
+    ```
+
+1. Change the `atlas.graph.index.search.solr.zookeeper-url` in the `conf/atlas-application.properties` file to point to zookeeper started by Solr. The default value is: `localhost:9983`. An example Solr URL is as follows:
+
+    ```conf
+    #Solr
+    #Solr cloud mode properties
+    atlas.graph.index.search.solr.zookeeper-url=localhost:9983
+    ```
+
+1. Start the Atlas Server from the Atlas home directory as follows:
+
+    ```sh
+    bin/atlas_start.py
+    ```
+
+    You should see the following output:
+
+    ```output
+    Starting Atlas server on host: localhost
+    Starting Atlas server on port: 21000
+    .........................
+    Apache Atlas Server started!!!
+    ```
+
+1. To ensure the server has started successfully, run the following command:
+
+    ```sh
+    # The default username and password for atlas is admin
+    curl -u username:password http://localhost:21000/api/atlas/admin/version
+    ```
+
+    You should see the following output:
+
+    ```output
+    {"Description":"Metadata Management and Data Governance Platform over Hadoop",    "Revision":"4cd215e1e2a04acbcd8afe6af95f43c4979202f1","Version":"2.3.0","Name":"apache-atlas"}
+    ```
+
+1. Run the quick start script using the following command:
+
+    ```sh
+    bin/quick_start.py
+    ```
+
+    When prompted for a username and password, enter `admin`. After it completes, you should see the following output:
+
+    ```output
+    Creating sample types:
+    Created type [DB]
+    Created type [Table]
+    .
+    Created type [Table_Columns]
+    Created type [Table_StorageDesc]
+
+    Creating sample entities:
+    Created entity of type [DB], guid: f87936f1-d620-4f70-88c1-471f30e95c68
+    .
+    Created entity of type [LoadProcess], guid: c4c0e468-5af9-474f-acc1-088747ebf199
+    Created entity of type [LoadProcessExecution], guid: a03dd462-437d-42ee-b30f-dd0b8f1e9413
+    Created entity of type [LoadProcessExecution], guid: 2fe2e4c2-04ac-4c34-9a14-67436428949d
+
+    Sample DSL Queries:
+    query [from DB] returned [3] rows.
+    query [DB] returned [3] rows.
+    .
+    query [from DataSet] returned [10] rows.
+    query [from Process] returned [3] rows.
+
+    Sample Lineage Info:
+    loadSalesMonthly(LoadProcess) -> sales_fact_monthly_mv(Table)
+    time_dim(Table) -> loadSalesDaily(LoadProcess)
+    sales_fact_daily_mv(Table) -> loadSalesMonthly(LoadProcess)
+    sales_fact(Table) -> loadSalesDaily(LoadProcess)
+    loadSalesDaily(LoadProcess) -> sales_fact_daily_mv(Table)
+    Sample data added to Apache Atlas Server.
+    ```
+
+1. You can also verify if the keyspaces `janusgraph` and `atlas_audit` got created using the [ycqlsh](../../admin/ysqlsh/#starting-ysqlsh) shell as follows:
+
+    ```output
+    DESC KEYSPACES;
+    ```
+
+1. Stop the Atlas server using the following command:
```

Comment:
Would put this cleanup in a separate section. Don't need numbered steps
## Clean up
You can stop the Atlas server using the following command:
etc